week5/sunny/qzone1/index.css

```diff
@@ -0,0 +1,266 @@
+body {
+  margin: 0;
+  background-image: url(beach.jpg);
+  background-size: cover;
+}
+
+.wrapper {
+  width: 100%;
+  height: 100vh;
+  max-width: 880px;
+  margin: auto;
+  position: relative;
+}
+
+.login-container {
+  position: absolute;
+  top: 95px;
+  width: inherit;
+}
+
+.login-container img {
+  width: 251px;
+  height: 98px;
+}
+
+.login-box {
+  position: absolute;
+  top: 100px;
+  right: 0;
+  background: #ffffff;
+  border: 2px solid #eeeeee;
+  border-radius: 4px;
+  font-family: Helvetica, Arial, 微软黑体, 华文黑体;
+}
+
+.login-header {
+  margin-bottom: 36px;
+  text-align: center;
+}
+
+.login-header h3 {
+  font-size: 20px;
+  font-weight: 500;
+  color: #333333;
+  margin-top: 5px;
+  margin-bottom: 16px;
+}
+
+.login-header p {
+  font-size: 16px;
+  color: #666666;
+  margin: 0;
+}
+
+.login-header a {
+  color: #3481cf;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+form {
+  width: 272px;
+  margin: 0 71px;
+}
+
+input {
+  width: inherit;
+  height: 42px;
+  margin-bottom: 13px;
```

Comment:
13?

Reply:
which means i dont need to follow exactly the numbers shows in inspect. i could use 12 or 14 instead?

Reply:
yes, always use even numbers